src/org/jitsi/impl/neomedia/rtcp/RTCPTCCPacket.java

```diff
@@ -539,21 +518,22 @@ public RTCPTCCPacket(long senderSSRC, long sourceSSRC,
             else if (seqDelta % 7 == 3)
```

Comment:
I don't know how to simplify this code. I can add some documentation if you think it's necessary.